nullaway/src/main/java/com/uber/nullaway/handlers/LibraryModelsHandler.java

```diff
@@ -233,6 +256,38 @@ public NullnessHint onDataflowVisitMethodInvocation(
     }
   }
 
+  /**
+   * Check if the given symbol is a field that is marked as nullable in any of our library models.
+   *
+   * @param symbol The symbol to check.
+   * @return True if the symbol is a field that is marked as nullable in any of our library models.
+   */
+  private boolean isNullableFieldInLibraryModels(@Nullable Symbol symbol) {
+    if (libraryModels.nullableFields().isEmpty()) {
+      // no need to do any work if there are no nullable fields.
+      return false;
+    }
+    if (symbol instanceof Symbol.VarSymbol && symbol.getKind().isField()) {
+      Symbol.VarSymbol varSymbol = (Symbol.VarSymbol) symbol;
+      Symbol.ClassSymbol classSymbol = varSymbol.enclClass();
+      if (classSymbol == null) {
+        // e.g. .class expressions
+        return false;
+      }
+      String fieldName = varSymbol.getSimpleName().toString();
+      String enclosingClassName = classSymbol.flatName().toString();
+      // check presence. This is a bit inefficient, but it's only used while indexing impact of
+      // making fields @Nullable on downstream dependencies. Also, the set of nullable fields is not
+      // expected to be large. We can optimize in future if needed.
+      return libraryModels.nullableFields().stream()
+          .anyMatch(
+              fieldRef ->
+                  fieldRef.getFieldName().equals(fieldName)
+                      && fieldRef.getEnclosingClassName().equals(enclosingClassName));
```

Comment:
One last question.  Why not just do `return libraryModels.nullableFields().contains(fieldRef(enclosingClassName,fieldName))`?  We don't care about performance too much, but that should be constant time assuming hashing, and also it seems clearer.

Reply:
Sure, make sense. 1728198fabe668b5f93929efbf1b5d517d6f3a3b